lib/libraries/default/spinner.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  render: function(){
+    var html = '',
+        artifacts = this.view._artifacts.spinner = {},
+        height = this.height() || 35,
+        offsetPadding = (height - 35) / 2,
+        prefixes = ['webkit', 'Moz', 'ms', 'O'],
+        radius = this.view._artifacts.radius = 0,
+        spinner;
+
+    // Build DOM element
+    html += '<div class="' + this.theme() + '">';
+    html +=   '<div class="keen-spinner-container" style="height: ' + height + 'px; padding-top: ' + offsetPadding + 'px;">';
+    html +=     '<div class="keen-spinner-indicator"></div>';
+    html +=   '</div>';
+    html += '</div>';
+    this.el().innerHTML = html;
+
+    spinner = this.el().querySelector('.keen-spinner-indicator');
+    if (spinner.style.animationName === undefined) {
+      radius = 0;
+      artifacts.interval = setInterval(function(){
+        radius = (radius === 350) ? 0 : radius + 10;
+        for (var i = 0; i < prefixes.length; i++) {
+          spinner.style[prefixes[i]] = 'rotate(' + artifacts.radius + 'deg)';
```

Comment:
Is there a way we could look at moving this stuff into the CSS? I feel like managing vendor prefixes here can become problematic. Managing in the CSS allows us to use things like `postcss` to automatically handle adding and deprecating all vendor prefixes.